examples/CMakeLists.txt

```diff
@@ -47,3 +47,8 @@ add_example(xlink_server xlink_server.cpp)
 add_example(xlink_server2 xlink_server2.cpp)
 # Boot firmware
 add_example(device_connect_reset device_connect_reset.cpp)
+
+# Local shared memory example
+add_example(xlink_server_local xlink_server_local.cpp)
+add_example(xlink_client_local xlink_client_local.cpp)
```

Comment:
Instead of doing "if unix" in source code, conditionally add these as examples IFF running on UNIX
Apply this elsewhere as well where needed (where we ifdef out a whole file - is better to make it "platform specific compiliation wise")